app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -248,15 +216,7 @@ fun LinkSpotifyButton() {
         Text(
             modifier = Modifier.testTag("linkSpotifyText"),
             text = "Link My Spotify Account",
-            style =
-                TextStyle(
-                    fontSize = 14.sp,
-                    lineHeight = 20.sp,
-                    fontFamily = FontFamily(Font(R.font.roboto)),
-                    fontWeight = FontWeight(500),
-                    color = PrimaryPurple,
-                    letterSpacing = 0.14.sp,
-                ))
+            style = TypographyBeatLink.bodyMedium)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
            style = MaterialTheme.typography.bodyMedium)
```